src/pages/references/clarity-js/clarity-js.md

```diff
@@ -0,0 +1,14 @@
+---
+title: '@blockstack/clarity'
```

Comment:
I'd suggest Clarity JS SDK

Reply:
Clarity JS SDK is the whole repository. There is an open issue how to properly name this package https://github.com/blockstack/clarity-js-sdk/issues/47